### DISCUSSION TITLE
Grove - Round Force Sensor FSR402 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#218] [Q&A]
# Grove - Round Force Sensor FSR402 | Seeed Studio Wiki

Grove - Round Force Sensor FSR402

https://wiki.seeedstudio.com/Grove-Round_Force_Sensor_FSR402/

**Comment by scottmclesly** [comment]
How do I decrease sensitivity of this sensor? I need to measure in ganges up to 6 lbs but I barely touch the sensor and it reads MAX value.

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
